heap_sort/HeapSort.java

```diff
@@ -0,0 +1,44 @@
+/*
+*  Following is the implementation of HeapSort algorithm
+*  It will sort the input array with time complexity O(N*logN)
+*  N being the size of array
+*/
+
+public class HeapSort {
+
+	public static void heapify(int[] arr, int size, int parent) {  // Build max-heap of subtree of parent index
```

Comment:
>Using max_heap will help a reader understanding the code more easily.

Comments help far much better 😄 
Plus if we do that, then in the master function too its occurrences will have to be changed. Also, `heapify()` implies a more general concept here that it can be converted to min or max heap with just changing two operators.